packages/vars/src/hooks/useBreakpoint.hook.ts

```diff
@@ -0,0 +1,98 @@
+import { useEffect, useState } from 'react';
+
+import breakpointsData from '../breakpoints.json';
```

Comment:
what is the point of using JSON in this case? why don't we use a JS object to store our breakpoints with the support of TypeScript